src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -275,6 +275,26 @@ private async Task RestartWorkerChannel(string runtime, string workerId)
             }
         }
 
+        private void AddOrUpdateErrorBucket(WorkerErrorEvent currentErrorEvent, TimeSpan threshold = default)
+        {
+            if (threshold == default)
```

Comment:
is threshold needed as  part of arguments?

Reply:
Maybe not